docs/datamap/bloomfilter-datamap-guide.md

```diff
@@ -26,7 +26,16 @@ Showing all DataMaps on this table
   SHOW DATAMAP
   ON TABLE main_table
   ```
-It will show all DataMaps created on main table.
+
+Enable Datamap
+  ```
+  SET carbon.datamap.visible.dbName.tableName.dataMapName = true
+  ```
+
+Disable Datamap
```

Comment:
Please add more description about it. Such as
```
We can disable a specific datamap during query to observe whether we can gain performance enhancement from it. This will only take effect current session.
```